pilot/pkg/model/gateway.go

```diff
@@ -465,9 +465,9 @@ func GetSNIHostsForServer(server *networking.Server) []string {
 	return sets.SortedList(sniHosts)
 }
 
-// CheckDuplicates returns all of the hosts provided that are already known
+// CheckGatewayHostsDuplicates returns all of the hosts provided that are already known
 // If there were no duplicates, all hosts are added to the known hosts.
-func CheckDuplicates(hosts []string, bind string, knownHosts map[string]string) []string {
+func CheckGatewayHostsDuplicates(hosts []string, bind string, knownHosts map[string]string) []string {
```

Comment:
Looks like there's no need to rename this function?

Reply:
This change does not make much sense, mainly for the convenience of review，two considerations:
1. function names are more readable and easy to reference
2. the code is referenced here for easy review

Reply:
the naming is verbose， It's better to stay the same 😆